packages/beta/src/api/templateGroups/templateGraphQlApi.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 Cognite AS
+
+import { BaseResourceAPI } from '@cognite/sdk-core';
+import { GraphQlResponse } from '../../types';
+
+export class TemplateGraphQlApi extends BaseResourceAPI<any> {
+  /**
+   * [Run a GraphQL query](https://pr-1202.specs.preview.cogniteapp.com/v1.json.html#operation/postApiV1ProjectsProjectTemplategroupsExternalidVersionsVersionGraphql)
+   *
+   * See https://graphql.org/learn/ for more info on how to construct a GraphQL query.
```

Comment:
Guess such comments breaks code snippets generation

Reply:
Hmm, I'm not sure if I follow. What do you mean breaks code snippets generation?

Reply:
If you'll look through the other comments to endpoints (`eventsApi.ts` for instance), all of them have typedoc comments explained how to use it (under ```js ... ``` section). All of them have a quite strict structure, cause we generate code-snippets for documentation purposes. Basically, you need to have a link to the API docs followed by js example of how to use the endpoint.

Reply:
You can take a look at how code snippets generate in the `scripts` folder of the project

Reply:
Wut, so you can't have description together with a title and JS snippet as this comment?

Reply:
Guess the idea behind this is to have all required information by link. But I also think that you can have it right above link sections without adding empty space between sections. I'll provide an example.

Reply:
As a workaround, you can specify the same comments in `js` section as well